javascript/src/HtmlEditorField.js

```diff
@@ -844,7 +844,14 @@ $.entwine('ss', function($) {
 				action = "update";
 			}
 
-			if(href.match(/^mailto:(.*)$/)) {
+            if(href.match(/^mailto:(.*)\?subject=(.*)$/)) {
```

Comment:
This would also fail for any links that were like `mailto:me@example.com?body=Body+here&subject=Subject+here`